src/librustdoc/html/render.rs

```diff
@@ -3558,6 +3560,62 @@ fn should_render_item(item: &clean::Item, deref_mut_: bool) -> bool {
     }
 }
 
+fn spotlight_decl(decl: &clean::FnDecl) -> String {
+    let mut out = Buffer::html();
+    let mut trait_ = String::new();
+
+    if let Some(did) = decl.output.def_id() {
+        let c = cache();
+        if let Some(impls) = c.impls.get(&did) {
```

Comment:
This isn't a good way to find applicable impls because it doesn't take into account things like generic parameters and mutability. For example:

![image](https://user-images.githubusercontent.com/7189418/87577858-14e9a400-c6c3-11ea-8e5f-14ca6fd8caf1.png)

and

![image](https://user-images.githubusercontent.com/7189418/87577906-2632b080-c6c3-11ea-9b55-bd5014915a01.png)

I'd imagine this will need to use a mechanism similar to how auto trait and blankets impls are found.

Reply:
Given that this is reintroducing an old feature that was previously accepted in this form, can we handle this as a followup?

Reply:
Like, I want to ladn this so we can gather more community feedback on this feature and grow it. It's a feature that existed in this form for a while and was recently removed, my priority is getting it back in so we can improve it.